app/assets/stylesheets/catarse_bootstrap/_main.scss

```diff
@@ -143,7 +144,8 @@ p {
 .section.full-height.hello-idea {
   padding-top: 108px;
   padding-bottom: 169px;
-  background-image: url('../images/bg-ideia.jpg');
+  background-image: url('../images/bg-icone-hello-6.jpg');
```

Comment:
actually it is right this way @devton 

we are using this class just in mockups (webflow.com sites). 

by the time we approve the mockup and code it we will have to duplicate this class on the _extra.sass   (because I can't use the assets in webflow with the catarse url)

do you remember that you had to do the same for the hero-image classes?